lib/drivers/dexcomDriver.js

```diff
@@ -645,6 +645,7 @@ module.exports = function (config) {
       }
       if (datum.annotation) {
         annotate.annotateEvent(cbg, datum.annotation);
+        cbg._forceUpdate = true;
```

Comment:
Yep, see https://github.com/tidepool-org/jellyfish/pull/58, which is already on staging.